redbeams/src/test/java/com/sequenceiq/redbeams/service/drivers/DriverFunctionsTest.java

```diff
@@ -23,14 +25,22 @@ public void setup() {
 
     @Test
     public void testDriverFunctions() {
-        driverFunctions.execWithDatabaseDriver("", DatabaseVendor.POSTGRES,
+        DatabaseConfig databaseConfig = new DatabaseConfig();
+        databaseConfig.setName("name");
+        databaseConfig.setDatabaseVendor(DatabaseVendor.POSTGRES);
+        databaseConfig.setConnectorJarUrl("");
+        driverFunctions.execWithDatabaseDriver(databaseConfig,
```

Comment:
I would rename driverFunctions to underTest and would leave a line above it blank to make it salient this line is the test.

Reply:
Sure thing.